go.mod

```diff
@@ -64,13 +67,16 @@ require (
 	gopkg.in/macaroon.v2 v2.0.0
 )
 
+require github.com/charmbracelet/x/ansi v0.1.4 // indirect
```

Comment:
Hmm, why is this its own `require` block? Shouldn't this be sorted in with the rest in the upcoming block for the indirect dependencies?

Reply:
(removed commit)